.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I agree, but it makes easier to contribute for VSCode users, since the folder has formatting and debugging settings. If i don't check it in, i am afraid to lose them with `git clean`... wonder how we can go about it.